<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessBoard</class>
 <widget class="QMainWindow" name="ChessBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChessBoard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>311</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#verticalWidget{
border-image:url(:/resources/images/b.jpg);}</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;对战计时：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_restart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>281</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>重新开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>281</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>悔棋</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_showstep">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>281</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>显示/隐藏 步数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_showmenu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>281</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主菜单</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;labelTextStep&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Btn_start">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_reset">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_about">
        <property name="text">
         <string>关于作者</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>651</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
